src/Dfc.CourseDirectory.Database/Pttcd/StoredProcedures/RefreshFindACourseIndexForTLevels.sql

```diff
@@ -23,8 +23,8 @@ BEGIN
 			1 DeliveryMode,  -- Classroom based,
 			CAST(0 AS bit) FlexibleStartDate,
 			t.StartDate,
-			0 Cost,
-			NULL CostDescription,
+			NULL Cost,
+			'T Levels are currently only available to 16-19 year olds.  Contact us for details of other suitable courses.' CostDescription,
```

Comment:
Perhaps better to just hard-code this output into the FAC details API rather than stuffing Sql with it? Unless it's required in the search index for anything?